packages/backend/src/routes/PostRouter.ts

```diff
@@ -105,13 +105,16 @@ async function createPost(req, res) {
     const currentEpoch = Number(await unirepContract.currentEpoch())
 
     // Parse Inputs
-    const { publicSignals, proof } = req.body
+    const { publicSignals, proof, title, content } = req.body
     const reputationProof = new ReputationProof(
         publicSignals,
         formatProofForSnarkjsVerification(proof)
     )
     const epochKey = reputationProof.epochKey.toString()
     const minRep = Number(reputationProof.minRep)
+    const hashedContent = ethers.utils.keccak256(
```

Comment:
I'm torn on this, maybe we should use poseidon so it's zk friendly?

Reply:
if we don't verify hash content in a zk proof we don't have to use poseidon
and now unirep social doesn't install the poseidon package (but there is `ethers` in the unirep social)
I prefer to use `keccak256`